src/ejabberd_cowboy.erl

```diff
@@ -300,5 +300,5 @@ trails_store() ->
       Trails = trails:trails(Mods),
       trails:store(Trails)
     catch Class:Exception ->
-        ?WARNING_MSG("Trails Call: [~p:~p/0] catched ~p:~p", [?MODULE, trails_store, Class, Exception])
+        io:format("WARNING! Trails Call: [~p:~p/0] catched ~p:~p~n", [?MODULE, trails_store, Class, Exception])
```

Comment:
The first approach was better. Let's use the macro instead of `io:format`. Did you notice any issues with the macro?

Reply:
Yep, the issue was in CI when I was use macros. CI was failed with error ` {{badmatch,{error,lager_not_running}}`

Reply:
Oh, it's because the `mod_websocket_SUITE` didn't have lager application started. You can start it by `application:ensure_all_started(lager)` in the `setup()` function in the suite and then stop it in the `teardown()`.

Reply:
Yes, thanks, done.